### ISSUE TITLE
Date formatting doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#4]
```
What steps will reproduce the problem?
1. Set a date preference.
2. Note that it's not respected

What is the expected output? What do you see instead?
Expected: Date format preference to be respected
Actual: System date format.

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `evancharlton` on 20 Nov 2010 at 12:15

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Mileage 3.0
```

Original comment by `evancharlton` on 30 Nov 2010 at 12:57
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `evancharlton` on 17 Apr 2011 at 8:21
- Changed state: **Released**
